db/seeds/affiliate.rb

```diff
@@ -1,3 +1,4 @@
-Affiliate.create(name: 'usagov', display_name: 'USA.gov')
-Affiliate.create(name: 'gobiernousa', display_name: 'GobiernoUSA.gov', locale: 'es')
-Affiliate.create!(name: 'i14y_enabled', display_name: 'I14y Affiliate', gets_i14y_results: true)
+# frozen_string_literal: true
+
+Affiliate.create(name: 'test_affiliate', display_name: 'Test Affiliate')
+Affiliate.create(name: 'spanish_affiliate', display_name: 'Test Spanish Affiliate', locale: 'es')
```

Comment:
I renamed the `usagov` and `gobiernousa` affiliates to be more generic for test purposes and to avoid confusion with the production `usagov` and `gobiernousa` affiliates.